### DISCUSSION TITLE
identifying marker genes with gene expression matrix

### DISCUSSION THREAD

**Discussion by mitchTwoTimes** [GreenleafLab/ArchR#1129] [Questions / Documentation]
Hello,

Can marker feature identification be used on the gene expression matrix of a multiome dataset? Here is an example function call:
```
markersGS.rna <- getMarkerFeatures(
  ArchRProj = proj,
  useMatrix = "GeneExpressionMatrix",
  groupBy = "Clusters_rna",
  bias = c("TSSEnrichment", "log10(nFrags)"),
  testMethod = "wilcoxon"
)
```

Are the ```bias``` arguments being used? Is there any harm to keeping them in for RNA data?

Thanks!

**Comment by rcorces** [comment]
yes this should work fine (for pretty much any matrix). The bias arguments are definitely used still. Every cell represented in the GeneExpressionMatrix would still have those attributes and thus you would select the null group of cells for comparison based on those bias groups. But you could obviously make them something different if relevant to your analysis.
